.cirrus.yml

```
@@ -7,11 +7,11 @@ flutter_docker_builder:
     # from https://flutter.dev/docs/development/tools/sdk/releases
     matrix:
     - DOCKER_TAG: latest
-      FLUTTER_VERSION: v1.5.4-hotfix.2
+      FLUTTER_VERSION: v1.7.8+hotfix.2
     - DOCKER_TAG: stable
-      FLUTTER_VERSION: v1.5.4-hotfix.2
+      FLUTTER_VERSION: v1.7.8+hotfix.2
     - DOCKER_TAG: beta
-      FLUTTER_VERSION: v1.6.3
+      FLUTTER_VERSION: v1.7.8+hotfix.2
     - DOCKER_TAG: dev
       FLUTTER_VERSION: v1.7.4
```

This should be also updated

```suggestion
      FLUTTER_VERSION: v1.7.11
```